### DISCUSSION TITLE
Render when using <component>

### DISCUSSION THREAD

**Discussion by FabienVINCENT** [nuxt/nuxt#29063] [Questions]
Hello,

I have a problem with the use of `<component>`.
In my application I use a component that I pass in a slot to nested components. It comes from the PrimeVue library.

I have a problem that I reproduced in a simpler example: [stackblitz](https://stackblitz.com/edit/github-wwtahf?file=components/Expand.vue,components/Main.vue,components/Child.vue)

I have a Main component in which I call a Child component. In the case where there is a named slot, I have the following strange behavior: as soon as the v-model is updated, it unmounts and remounts the <component>.

Am I using it wrong? (knowing that the Child.vue would be the library and therefore that I do not have control over it?

**Comment by FabienVINCENT** [comment]
The problem is still present in new versions of nuxt 4, can someone help me

**Reply by OrbisK** [reply]
I think this is not related to nuxt. I think this is related to vue
